src/components/charts/horizontal-bar-chart.js

```diff
@@ -31,57 +31,55 @@ export default function HorizontalBarChart({
     .range([width - totalXMargin, 0])
 
   return (
-    <div>
-      <svg className={chartStyles.chart} viewBox={`0 0 ${width} ${height}`}>
-        <g transform={`translate(${marginLeft} ${marginTop})`}>
-          {xScale.ticks(xTicks).map(tick => (
-            <g key={tick}>
-              <text
-                className={`${chartStyles.label} ${chartStyles.xTickLabel}`}
-                x={245 - xScale(tick)}
-                y={height - marginBottom}
-              >
-                {formatTick(tick)}
-              </text>
-              <line
-                className={chartStyles.gridLine}
-                x1={xScale(tick)}
-                x2={xScale(tick)}
-                y1={0}
-                y2={height - totalYMargin}
-              />
-            </g>
-          ))}
-        </g>
-
-        <g transform={`translate(0, ${marginTop})`}>
-          {data.map(d => (
-            <svg
-              y={yScale(d.name) + 11}
-              x={marginLeft - 10}
-              className={chartStyles.yTickLabel}
-              key={d.name}
+    <svg className={chartStyles.chart} viewBox={`0 0 ${width} ${height}`}>
+      <g transform={`translate(${marginLeft} ${marginTop})`}>
+        {xScale.ticks(xTicks).map(tick => (
+          <g key={tick}>
+            <text
+              className={`${chartStyles.label} ${chartStyles.xTickLabel}`}
+              x={245 - xScale(tick)}
+              y={height - marginBottom}
             >
-              <text className={chartStyles.label} textAnchor="end">
-                {`${d.name}`}
-              </text>
-            </svg>
-          ))}
-        </g>
-
-        <g transform={`translate(${marginLeft}, ${marginTop})`}>
-          {data.map(d => (
-            <rect
-              key={d.name}
-              x={0}
-              y={yScale(d.name)}
-              height={yScale.bandwidth()}
-              width={xScale(0) - xScale(d.value)}
-              fill={fill}
+              {formatTick(tick)}
+            </text>
+            <line
+              className={chartStyles.gridLine}
+              x1={xScale(tick)}
+              x2={xScale(tick)}
+              y1={0}
+              y2={height - totalYMargin}
             />
-          ))}
-        </g>
-      </svg>
-    </div>
+          </g>
+        ))}
+      </g>
+
+      <g transform={`translate(0, ${marginTop})`}>
+        {data.map(d => (
+          <svg
```

Comment:
Why embed another `<svg>` inside of the existing `<svg>`?

Reply:
Having the nested element allowed me to keep the containers left aligned, but text right justified with `textAnchor="end"`. Was [following example here](https://developer.mozilla.org/en-US/docs/Web/SVG/Attribute/text-anchor).

From what I've read, nesting `<svg>` is allowable, and helps mimic relative positioning:

https://www.sarasoueidan.com/blog/nesting-svgs/
http://tutorials.jenkov.com/svg/svg-element.html

I tried using a `<g>` element but it clobbered the layout, stopped honoring the y position, and I couldn't figure out why:

<img width="500" alt="Screen Shot 2020-04-20 at 4 37 05 PM" src="https://user-images.githubusercontent.com/63169602/79797483-51586e80-8325-11ea-93cc-c3a1df341e6f.png">

Since `<svg>` worked, I left it at that. But welcome other ideas!

Reply:
Oh, woah! Thanks for pointing me towards this, that's a pretty powerful technique I had no idea about.